packages/studio/src/components/ErrorComponentPreview.tsx

```diff
@@ -22,11 +31,36 @@ export default function ErrorComponentPreview(props: {
     setTooltipOpen(false);
```

Comment:
actually, could we just rename this function to `closeTooltip` or something so it can be used for `onMouseLeave` and the listeners below?

Reply:
Yup can do, was debating that haha